challenge_final/src/challenge_final/navigate_to_and_grab_rack.py

```diff
@@ -132,6 +132,6 @@ def __init__(self, robot, rack_id, rack_navigation_area):
 
 if __name__ == '__main__':
     rospy.init_node(os.path.splitext("test_" + os.path.basename(__file__))[0])
-    hero = Hero()
-    hero.reset()
-    NavigateToAndGrabRack(hero, "rack", "in_front_of").execute()
+    robot_instance = get_robot("hero")
```

Comment:
I don't like this. Do you really think that using `robot` would be too unclear/confusing?

Reply:
The problem is that this is in the same file. As a result, you get a warning that names shadow a variable in the outer scope (which I consider bad practice). Therefore, I want to use a different name and this seems suitable to me.